src/libraries/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -3456,4 +3456,7 @@ Usage: dotnet {0} [--assembly &lt;assembly file path&gt;] [--type &lt;type name&
   <data name="CompilingScriptsNotSupported" xml:space="preserve">
     <value>Compiling JScript/CSharp scripts is not supported</value>
   </data>
+  <data name="XPath_UnexpectedTokens" xml:space="preserve">
+    <value>XPath failed to match an elements</value>
```

Comment:
Could you fix the grammar here? Was wrong before of course. ("XPath failed to match any 
```suggestion
    <value>XPath failed to match any elements</value>
```